internal/admin/dashboard/templates/index.html

```
@@ -384,27 +384,45 @@ <h3 x-text="guardrailFormMode === 'edit' ? 'Edit Guardrail' : 'Create Guardrail'
+                                <small class="alias-form-hint" x-show="field.help" x-text="field.help"></small>
+                            </label>
                         </template>
-                        <template x-if="field.input !== 'select' && field.input !== 'textarea'">
-                            <input :type="field.input || 'text'" class="filter-input" :placeholder="field.placeholder || ''" :value="guardrailFieldValue(field)" @input="setGuardrailFieldValue(field, $event.target.value)">
+                        <template x-if="field.input === 'checkboxes'">
+                            <div class="alias-form-field alias-form-field-wide">
+                                <span x-text="field.label"></span>
+                                <div class="execution-plan-feature-toggles">
+                                    <template x-for="option in field.options" :key="field.key + '-' + option.value">
+                                        <label class="execution-plan-feature-toggle">
+                                            <input type="checkbox" :checked="guardrailArrayFieldSelected(field, option.value)" @change="toggleGuardrailArrayFieldValue(field, option.value, $event.target.checked)">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)